docs/plugins/discovery-ec2.asciidoc

```diff
@@ -110,10 +110,10 @@ The available values are:
 * `cn-north` (`cn-north-1`)
 
 [[discovery-ec2-usage-signer]]
-===== EC2/S3 Signer API
+===== EC2 Signer API
 
-If you are using a compatible EC2 or S3 service, they might be using an older API to sign the requests.
-You can set your compatible signer API using `cloud.aws.signer` (or `cloud.aws.ec2.signer` and `cloud.aws.s3.signer`)
+If you are using a compatible EC2 service, they might be using an older API to sign the requests.
+You can set your compatible signer API using `cloud.aws.signer` (or `cloud.aws.ec2.signer`)
 with the right signer to use. Defaults to `AWS4SignerType`.
```

Comment:
I saw in https://github.com/elastic/elasticsearch-cloud-aws/issues/242 that this is wrong actually.
And the code says that default is actually `null` or not specified... I would remove here `Defaults to ` or replace it with `Defaults to not specified`